lib/takeaway.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'menu'
+require_relative 'order'
+
+class TakeAway 
+
+  
+
+  def initialize 
+    @menu = Menu.new
+    @order = Order.new
+    @finish = false
+  end 
+
+  def show_menu
+    @menu.print_menu
+  end
+
+  def add(input, qua = 1)
+    return puts "I'm afraid we don't have any #{input}" unless valid?(input)
```

Comment:
Instead of `return puts "I'm afraid we don't have any #{input}" unless valid?(input)` you can just drop the puts and `return "I'm afraid we don't have any #{input}" unless valid?(input)` or even better surface an error here with `raise “I’m afraid we don't have any #{input}" unless valid?(input)`.